app/mutations/userDisable.graphql

```diff
@@ -0,0 +1,5 @@
+mutation UserDisable($id: ID!) {
+  disableUser: UserDisable(id: $id) {
```

Comment:
The mutation name is `userDisable`, fix the casing.

Reply:
oh sorry for that typo